moped-editor/src/views/projects/projectView/NoteInputQuill.js

```diff
@@ -47,6 +52,23 @@ const CommentInputQuill = ({
     ref?.current.focus();
   }, []);
 
+  /**
+   * Defines the NewNoteTypeButton with a toggle style-change behavior.
+   * @param {Object} props
+   * @return {JSX.Element}
+   * @constructor
+   */
+  const NewNoteTypeButton = (props) => (
+    <Button
+      color="primary"
+      className={classes.showButtonItem}
+      variant={newNoteType === props.noteTypeId ? "contained" : "outlined"}
+      onClick={() => setNewNoteType(props.noteTypeId)}
+    >
+      {props.children}
+    </Button>
```

Comment:
I love that you use `props.children` here. I _never_ think to do this and it makes for such cleaner code!

That said...would you mind switching this to the [Radio component](https://mui.com/material-ui/react-radio-button/#direction) with a horizontal layout?

Like this (but plz ignore dark mode):

<img width="292" alt="Screenshot 2023-10-17 at 8 58 19 AM" src="https://github.com/cityofaustin/atd-moped/assets/14793120/0f19c750-54b9-4c76-afeb-f8df6ddaf190">

I'm sorry for not having this spec available for you when you started the work!

Reply:
oops idk why i assumed radio buttons were just the same kind of buttons as the ones above the comments section. ill fix that! it will be better having this set of buttons be different from the ones for sorting the notes.